tests/tests.js

```diff
@@ -1,45 +1,141 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const {
+  afterEach,
+  beforeEach,
+  describe,
+  it
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+chai.use(sinonChai)
+const {
+  expect
+} = chai
+
+describe('formatPrice', () => {
+  it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {
+    const x = 15.1566663
+    const formattedPrice = pricing.formatPrice(15.1566663)
 
-    expect(price).to.equal(39.37)
+    expect(formattedPrice).to.equal(15.15)
   })
+})
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
-        { role: 'sp', coverage: 75000 },
-      ],
+describe('commuterTrain', () => {
+  it('calculates the employee fare on the commuter train, minus the contribution from the employer', () => {
+    const commuterTrain = {
+      price: 84.75 - (products.employerContribution.contribution)
     }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
 
-    expect(price).to.equal(71.09)
+    expect(commuterTrain).to.equal(9.75)
   })
+})
 
-  it('returns the price for a disability product for an employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee']
+describe('commuterParking', () => {
+  it('calculates the fee payable by the employee for parking, short of the employer contribution', () => {
+    const commuterParking = {
+      price: 250 - (products.employerContribution.contribution)
     }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
-
-    expect(price).to.equal(22.04)
+    expect(commuterParking).to.equal(175)
   })
```

Comment:
These tests don't actually call any functions. They should be unit tests for whatever function you will create to calculate the commuter plan code. Remember that you were provided with the selectedOptions format in class so that should be part of this setup. We'd also expect there to be integration tests of `calculateProductPrice` since it will be able to handle commuter products now.

```JavaScript
const selectedOptions = {
    benefit: 'train'
}
```